src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -1,18 +1,43 @@
 package frc.robot.subsystems.intake;
 
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.RobotConstants.IntakeConstants;
 
 public class Intake extends SubsystemBase {
-  /** Creates a new ExampleSubsystem. */
-  public Intake() {}
+  private final CANSparkMax m_rollerMotor; // right motor (Rollers)
+
+  private final RelativeEncoder m_rollerEncoder; // Roller Relative Encoder
```

Comment:
don't need extra lines, when you need to get position or velocity you can just run motor.getEncoder() etc and your code will be more compact

Reply:
Removed encoder in "removed encoder" commit